hack/boilerplate/boilerplate.go.txt

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
do we really need any other bolierplate than this one?

Reply:
seems not. However i copied all from upstream just to have minimum differences wherever possible.